stripe/__init__.py

```diff
@@ -186,22 +186,20 @@ def __getattr__(name):
 # API resources
 
 # The beginning of the section generated from our OpenAPI spec
-from stripe import (
-    apps as apps,
-    billing_portal as billing_portal,
-    checkout as checkout,
-    climate as climate,
-    financial_connections as financial_connections,
-    identity as identity,
-    issuing as issuing,
-    radar as radar,
-    reporting as reporting,
-    sigma as sigma,
-    tax as tax,
-    terminal as terminal,
-    test_helpers as test_helpers,
-    treasury as treasury,
-)
+import stripe.apps as apps
```

Comment:
This won’t work; mypy doesn’t treat this as an explicit reexport like it does `from stripe import apps as apps`.

The only reason `from stripe import apps as apps` appeared not to be working was that a bug in the module-level `__getattr__` was causing module-level variables of these names (indeed, every name) to be defined as `None` rather than undefined:

- #1159

Reply:
Yep, you are right. Thank you @andersk, we'll merge your change first and this PR after.